helix-core/src/main/java/org/apache/helix/manager/zk/zookeeper/ZkClient.java

```diff
@@ -521,6 +543,11 @@ public String create(final String path, Object data, final CreateMode mode)
    */
   public String create(final String path, Object datat, final List<ACL> acl, final CreateMode mode)
       throws IllegalArgumentException, ZkException {
+    return create(path, datat, acl, mode, null);
+  }
+
+  private String create(final String path, Object datat, final List<ACL> acl, final CreateMode mode,
```

Comment:
need JavaDoc to explain what is usage of sessionId here.

Reply:
Agreed. I've added detailed java doc for this api in my local branch.